### PR TITLE
[test] Add todi configuration and adapted ce tests

### PR DESCRIPTION
**Pull request by teojgo**

### CODE REVIEW

File: checks/system/ce/ce_import_run_image.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ class enroot_import_image_dockerhub(enroot_import_image):
 
 class enroot_import_image_ngc(enroot_import_image):
     image = variable(
-        str, value='docker://nvcr.io#nvidia/k8s/cuda-sample:nbody'
+        str, value='docker://nvcr.io#nvidia/hpc-benchmarks:24.03'
```
Comment (jgphpc) [inline checks/system/ce/ce_import_run_image.py:39]: missing space line 22

Comment (jgphpc) [reply checks/system/ce/ce_import_run_image.py:39]: +  lines 47 and 67

**Review by jgphpc — APPROVED** [review]
```
[       OK ] (1/4) enroot_import_image_dockerhub ~todi /4c119ed3 @todi:normal+builtin
[       OK ] (2/4) RunJobCE /ad4db742 @todi:normal+builtin
[       OK ] (3/4) enroot_import_image_ngc ~todi /4d1b273e @todi:normal+builtin
[       OK ] (4/4) RunNVGPUJobCE /80e3f3c4 @todi:normal+builtin
```

File: checks/system/ce/ce_import_run_image.py [hunk 2]
```diff
@@ -91,17 +91,14 @@ def set_image_path(self):
     @run_before('run')
     def set_executable_opts(self):
         self.executable_opts = [
-            '-benchmark', '-fp64',
-            f'-numbodies={self.num_bodies_per_gpu * self.num_gpus_per_node}',
-            f'-numdevices={self.num_gpus_per_node}'
+            f'-d0', f'-n{self.stream_array_size}',
```
Comment (jgphpc) [inline checks/system/ce/ce_import_run_image.py:94]: not critical, but I would move @run_after('setup') = line 86 up before  @run_before('run') = line 78

**Review by ekouts — APPROVED** [review]

File: checks/system/ce/ce_import_run_image.py (hunk 1 above)
Comment (teojgo) [reply checks/system/ce/ce_import_run_image.py:39]: > * lines 47 and 67

What's the problem there?

Comment (jgphpc) [reply checks/system/ce/ce_import_run_image.py:39]: - pycodestyle checks/system/ce/ce_import_run_image.py
```
checks/system/ce/ce_import_run_image.py:22:24: E225 missing whitespace around operator
checks/system/ce/ce_import_run_image.py:47:25: E261 at least two spaces before inline comment
checks/system/ce/ce_import_run_image.py:67:25: E261 at least two spaces before inline comment
```

**Pull request merged**